charts/fluid-dataloader/juicefs/templates/configmap.yaml

```diff
@@ -23,9 +23,9 @@ data:
 
           for((j=0;j<${#paths[@]};j++)) do
             echo -e "juicefs warmup on $pod ${paths[j]} starts"
-            /usr/local/bin/kubectl -n $ns exec -it $pod -- $COMMAND
-            /usr/local/bin/kubectl -n $ns exec -it $pod -- juicefs warmup $MOUNTPATH${paths[j]}
-            /usr/local/bin/kubectl -n $ns exec -it $pod -- umount $MOUNTPATH
+            /usr/local/bin/kubectl -n $ns exec -it $pod --request-timeout=10s -- $COMMAND
+            /usr/local/bin/kubectl -n $ns exec -it $pod --request-timeout=1m -- juicefs warmup $MOUNTPATH${paths[j]}
```

Comment:
I think the timeout of warmup behavior should be configurable. By the way, is 1Minute enough for juicefs warm scenario?

Reply:
done